src/Stripe.net/Entities/OAuth/OAuthToken.cs

```diff
@@ -4,31 +4,62 @@ namespace Stripe
 
     public class OAuthToken : StripeEntity<OAuthToken>
     {
-        [JsonProperty("token_type")]
-        public string TokenType { get; set; }
-
-        [JsonProperty("scope")]
-        public string Scope { get; set; }
+        /// <summary>
+        /// <para>
+        /// The access token you can use to make requests on behalf of this Stripe account. Use it
+        /// as you would any Stripe secret API key.
+        /// </para>
+        /// <para>
+        /// This key does not expire, but may be revoked by the user at any time (you'll get a
+        /// <c>account.application.deauthorized</c>
+        /// <see href="https://stripe.com/docs/api/events/types#event_types-account.application.authorized">webhook event</see>
+        /// when this happens).
+        /// </para>
+        /// </summary>
+        [JsonProperty("access_token")]
```

Comment:
Can we mark this as deprecated in favour of the `Stripe-Account` header? We really should not recommend that anyone uses this property.

Reply:
👍